projects/custody-key-dates-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/custody/date/KeyDate.kt

```diff
@@ -48,4 +42,6 @@ class KeyDate(
             it.version = version
```

Comment:
Don't we need to un-delete the record as well?
```suggestion
            it.version = version
            it.softDeleted = false
```

Reply:
the entity set the soft delete flag to false by default.  however, i will make the change to make this clearer

Reply:
done